apps/frontend/src/libs/types/types.ts

```diff
@@ -5,3 +5,4 @@ export {
 	type ValidationSchema,
 	type ValueOf,
 } from "@git-fit/shared";
+export { type ReactNode } from "react";
```

Comment:
You can use just React.ReactNode where you need. It's not necessary to add it to types